aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -217,7 +232,41 @@ Tensor& cumprod_out(Tensor& result, const Tensor& self, int64_t dim, c10::option
   return result;
 }
 
+std::tuple<Tensor&, Tensor&> cummax_out(Tensor& values, Tensor& indices, const Tensor& self, int64_t dim) {
+  check_scalar_type_device_layout_equal(values, self);
+  check_scalar_type_device_layout_equal(indices, at::empty({}, self.options().dtype(at::kLong)));
```

Comment:
I see this was discussed in the other PR but the suggestion given was incorrect: this call will be allocate memory because a tensor of shape "{}" has 1 element.  "{0}" has 0 elements.

Reply:
Ho really? My bad !
@anjali411 can you send a PR to fix this?

Reply:
I'll fix it with cummin here https://github.com/pytorch/pytorch/pull/32238